src/network/WiFiHandler.h

```diff
@@ -0,0 +1,76 @@
+/*
+    SlimeVR Code is placed under the MIT license
+    Copyright (c) 2022 TheDevMinerTV
+
+    Permission is hereby granted, free of charge, to any person obtaining a copy
+    of this software and associated documentation files (the "Software"), to deal
+    in the Software without restriction, including without limitation the rights
+    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+    copies of the Software, and to permit persons to whom the Software is
+    furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in
+    all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+    THE SOFTWARE.
+*/
+
+#ifndef NETWORK_WIFIHANDLER_H
+#define NETWORK_WIFIHANDLER_H
+
+#include "configuration/WiFiCredential.h"
+#include "logging/Logger.h"
+
+#ifdef ESP8266
+    #include <ESP8266WiFi.h>
+    #include <ESP8266WiFiMulti.h>
+    #define WiFiMulti ESP8266WiFiMulti
+#else
+    #include <WiFi.h>
+    #include <WiFiMulti.h>
+#endif
+
+namespace SlimeVR {
+    namespace Network {
+        class WiFiHandler {
+        public:
+            void setup();
+            void update();
```

Comment:
Add comment, what does this do?

Reply:
This function gets called every iteration of `main.cpp`'s `loop()` and updates the `WiFiMulti instance.

Reply:
Makes sense, I've added a suggestion